app/controllers/resources_controller.rb

```diff
@@ -150,7 +150,15 @@ def get_fileset_uuid(external_id)
   def metadata_for_blob(blob, file)
     file.delete(:externalIdentifier)
     file[:size] = blob.byte_size
-    file[:hasMimeType] = blob.content_type || 'application/octet-stream'
+    # Invalid JSON files uploaded through H2 will trigger 400 errors in sdr-api since they are parsed
+    # and rejected.  The work around is to change the content_type in H2 for any JSON files to something
+    # specific that will be changed back to application/json after upload is complete.
+    # There is a similar change in H2.  See https://github.com/sul-dlss/happy-heron/issues/3075
```

Comment:
```suggestion
    # Invalid JSON files uploaded for deposit with a JSON content type will trigger 400 errors in sdr-api since they are
    # parsed as JSON and rejected.  The work around is to change the content_type in the request for uploads like this to something
    # specific that will be changed back to application/json after upload is complete.
    # There is a corresponding translation in sdr-client.  See https://github.com/sul-dlss/happy-heron/issues/3075
```

for your consideration, a slight tweak to the wording for more specificity (and to update it now that the content-type translation is being done in sdr-client)